api/src/main/java/graphql/nadel/ServiceExecutionHydrationDetails.java

```diff
@@ -1,15 +1,21 @@
 package graphql.nadel;
 
+import graphql.schema.FieldCoordinates;
+
 /**
  * This contains details about a service hydration call when a {@link ServiceExecution} is invoked.
  */
 public class ServiceExecutionHydrationDetails {
     private final int timeout;
     private final int batchSize;
+    private final FieldCoordinates hydrationSourceField;
+    private final Service hydrationSourceService;
```

Comment:
So question to the reviewers - should we send in JUST the service name or the whole Service

`Service` is public API and it has more than name - even though we only need the service name for metrics in AGG land.

I lean towards the whole Service object but would consider just the name

Reply:
The field coordinates would allow us to tag hydration metrics by source field as well as by source service

Reply:
`Service` is good imo